src/logics/Theory.C

```diff
@@ -58,13 +58,13 @@ namespace{
     }
 }
 
-//
-// Given coll_f = (R_1 /\ ... /\ R_{curr-1}) /\ (U_1 /\ ... /\ U_{curr-1}) /\
-// P_{curr}, compute U_{curr}.  Place U_{curr} to frame and simplify
+/*
+// Given coll_f = (R_1 /\ ... /\ R_{curr-1}) /\ (U_1 /\ ... /\ U_{curr-1}) /\ P_{curr},
```

Comment:
Maybe clean this comment so as to not have the //'s

Reply:
I removed the additional /* */ since other comments in that file seems to use //.
The only fix needed was for the comment not to end the line with \